warehouse/query-core/pom.xml

```diff
@@ -235,6 +235,12 @@
             <version>${version.accumulo}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.bouncycastle</groupId>
+            <artifactId>bcprov-jdk15on</artifactId>
+            <version>1.60</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Needed for test, and no longer comes in transitively via type-utils' geo dependencies